snuba/replacer.py

```diff
@@ -99,7 +99,7 @@ class Replacement:
 
 
 class ReplacerWorker(AbstractBatchWorker):
-    def __init__(self, clickhouse: ClickhousePool, dataset: Dataset, metrics=None) -> None:
+    def __init__(self, clickhouse: ClickhousePool, dataset: EventsDataset, metrics=None) -> None:
```

Comment:
Only `EventsDataset` has the `get_required_columns` method used on line 107 so this is required to type check (see description.)